test/testcafe/framework/page-objects/ChallengeEmailPageObject.js

```diff
@@ -8,16 +8,33 @@ export default class ChallengeEmailPageObject extends ChallengeFactorPageObject
     super(t);
   }
 
-  resendEmailView() {
-    return this.form.getElement(RESEND_EMAIL_VIEW_SELECTOR);
+  resendEmailViewText() {
+    if (userVariables.v3) {
+      return this.form.getErrorBoxText();
```

Comment:
Since this isn't actually an error but a warning type of text, I would update this function name (or create a new function to minimize changes) to something like `getAlertBoxText`